### PR TITLE
Doc: Use latest rosbridge_suite for working ActionLib Widget

### PR DESCRIPTION
**Pull request by plieningerweb**
Why? In current rosbridge_suite master the ActionLib Service is not yet implemented. Only in develop branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +29,30 @@ Until now, it is being developed and tested using Google Chrome browser.
     $ rosdep install web_video_server
     ```
 
-3. Launch rosbridge websocket server and web video server
+
+4. Install latest development version of rosbrige_server [you can skip this, but then the ActionLib Widget is not working]
```
Comment (marcoarruda) [inline README.md]: This is step 3

File: README.md [hunk 2]
```diff
@@ -29,13 +29,30 @@ Until now, it is being developed and tested using Google Chrome browser.
...
+    $ # go to your catkin workspace /src
+    $ git clone -b develop https://github.com/RobotWebTools/rosbridge_suite.git
+    $ cd ..
+    $ source devel/setup.bash
+    $ catkin_make
+    $  # edit src/rosbridge_suite/rosbridge_server/launch/rosbridge_websocket.launch
+    $  # replace each glob args line of  <arg name="topics_glob" default="[]" /> 
+    $  # with <arg name="topics_glob" default="[*]" /> 
+    $  # if you do not add the "*" no topic will be shown!
+    $ roslaunch rosbridge_server rosbridge_websocket.launch
```
Comment (marcoarruda) [inline README.md]: Remove this line and next

File: README.md [hunk 3]
```diff
@@ -29,13 +29,30 @@ Until now, it is being developed and tested using Google Chrome browser.
     $ rosdep install web_video_server
     ```
 
-3. Launch rosbridge websocket server and web video server
+
+4. Install latest development version of rosbrige_server [you can skip this, but then the ActionLib Widget is not working]
+    ```sh
+    $ # go to your catkin workspace /src
+    $ git clone -b develop https://github.com/RobotWebTools/rosbridge_suite.git
```
Comment (marcoarruda) [inline README.md]: Better cloning my repository (https://github.com/marcoarruda/rosbridge_suite), where I commit the updated for ROSWeb

File: src/widgets/camera_viewer/index.hbs [hunk 4]
```diff
@@ -2,9 +2,9 @@
   data-max-height="600" data-height="360" data-width="480">
 
   <!--  META Tags to describe topics subscription -->
-  <meta data-ros-topic="1" data-ros-topic-id="1" data-ros-topic-desc="Topic to subscribe" data-ros-topic-type="sensor_msgs/Image"
+  <meta data-ros-topic="1" data-ros-topic-id="1" data-ros-topic-desc="Topic to subscribe" data-ros-topic-type="sensor_msgs/Image|sensor_msgs/CompressedImage"
```
Comment (marcoarruda) [inline src/widgets/camera_viewer/index.hbs]: Not sure if it is going to work, have you tested it?

**Comment by marcoarruda** [comment]
Thanks!
Could you apply the changes I've suggested?

File: README.md (hunk 1 above)
Comment (plieningerweb) [inline README.md]: I dont get it...
first install normal version, then install git dev version (so all dependencies are there)

File: README.md (hunk 3 above)
Comment (plieningerweb) [inline README.md]: makes sence

File: README.md (hunk 2 above)
Comment (plieningerweb) [inline README.md]: why?

File: src/widgets/camera_viewer/index.hbs (hunk 4 above)
Comment (plieningerweb) [inline src/widgets/camera_viewer/index.hbs]: compressed image is not displayed, as the js viewer is not updated yet

**Comment by marcoarruda** [comment]
Verified and accepted, thanks!

**Pull request merged**